cmake/OpenCVDownload.cmake

```diff
@@ -67,6 +115,8 @@ function(ocv_download)
     set(${DL_STATUS} TRUE PARENT_SCOPE)
   endif()
 
+  ocv_cmake_hook(OCV_DOWNLOAD_MIRROR_${OPENCV_DOWNLOAD_MIRROR_ID})
```

Comment:
> `OCV_DOWNLOAD_MIRROR_${OPENCV_DOWNLOAD_MIRROR_ID}`
> `_${OPENCV_DOWNLOAD_MIRROR_ID}`

We do not need dynamic suffix in the hook name.

Just `OPENCV_DOWNLOAD_PRE`.

---

Or we should have to `include(...cmake/mirrors/${__HOST}.cmake")` files which would register known hooks.

and use here `ocv_cmake_hook(OCV_DOWNLOAD_PACKAGE_${DL_ID})`

Reply:
`OPENCV_DOWNLOAD_PRE` is used.